__tests__/buildTranslationFiles.spec.ts

```diff
@@ -9,35 +9,72 @@ import { resolveProjectBasePath } from '../src/utils/resolve-project-base-path';
 
 const sourceRoot = '__tests__';
 
-const m = 'missing';
-
-function gKeys(len: number, prefix?: string) {
-  let expected = {};
-  for (let i = 1; i <= len; i++) {
-    expected[prefix ? `${prefix}.${i}` : i] = m;
+const defaultValue = 'missing';
+
+function getRangeKeys({
+  start = 1,
+  len,
```

Comment:
If you decide to use `len` than the loop should be:
```ts
for (let i = start; i <= start + len; i++) {
```
Currently, you are using it as `end`, either rename to `end`, or use `len` as intended.